data_steward/gcloud/gcs/__init__.py

```diff
@@ -166,18 +170,21 @@ def _get_hpo_bucket_id(self, hpo_id: str) -> str:
         or (2) multiple records are found in the lookup table.
         """
         service = os.environ.get('GAE_SERVICE', 'default')
+        LOGGER.info(f"GAE_SERVICE: {service}")
+
+        hpo_bucket_query = JINJA_ENV.from_string(
+            SELECT_BUCKET_NAME_QUERY).render(
+                project_Id=self.project,
+                dataset_id=LOOKUP_TABLES_DATASET_ID,
+                table_id=HPO_ID_BUCKET_NAME_TABLE_ID,
+                hpo_id=hpo_id,
+                service=service)
 
-        hpo_bucket_query = JINJA_ENV.from_string(SELECT_ALL_QUERY).render(
-            project_id=self.project,
-            dataset_id=LOOKUP_TABLES_DATASET_ID,
-            table_id=HPO_ID_BUCKET_NAME_TABLE_ID)
+        LOGGER.info(f"Running: {hpo_bucket_query}")
 
         result_df: DataFrame = query(hpo_bucket_query)
-        condition_hpo_id = (result_df['hpo_id'] == hpo_id)
-        condition_service = (result_df['service'] == service)
-        result_filtered = result_df[condition_hpo_id & condition_service]
 
-        if result_filtered['bucket_name'].count() != 1:
+        LOGGER.info(f"hpo_")
```

Comment:
@ksdkalluri what is this logging?

Reply:
I am fixing it, didn't realize that was in the code.